package.json

```diff
@@ -35,22 +36,25 @@
     "file-loader": "3.0.1",
     "flow-bin": "0.54.1",
     "lqip-loader": "^2.2.0",
-    "webpack-dev-server": "^3.1.14"
+    "webpack-dev-server": "^3.7.2"
   },
   "dependencies": {
     "@babel/cli": "7.2.3",
     "@babel/core": "^7.3.4",
+    "@babel/plugin-proposal-class-properties": "^7.5.5",
+    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
+    "@babel/plugin-transform-runtime": "^7.5.5",
     "@babel/preset-react": "^7.0.0",
-    "@rails/webpacker": "^4.0.2",
+    "@rails/webpacker": "^4.0.7",
```

Comment:
Again, let's keep this one up-to-date. In this case, two for one deal (one yarn alert fixed).